README.md

```diff
@@ -6,104 +6,9 @@
-  * **Hit enter and select default options for any configuration popups**
-1. Upgrade npm to 2.0. If `npm -v` says less than 2.0 then
-  * `sudo add-apt-repository ppa:chris-lea/node.js  `
-  * `sudo apt-get update`
-  * `sudo apt-get install nodejs`
-1. Either A. Install Ruby 2.2 from OS packages; B. Setup `rbenv`; or C. Setup `rvm`
-    - A. OS packages (from the [Brightbox Ubuntu PPA](https://www.brightbox.com/docs/ruby/ubuntu/)):
-        - `sudo apt-get install software-properties-common`
-        - `sudo apt-add-repository ppa:brightbox/ruby-ng`
-        - `sudo apt-get update`
-        - `sudo apt-get install ruby2.2 ruby2.2-dev`
-    - B. `rbenv`: ([instructions](https://github.com/sstephenson/rbenv#installation))
-        1. Install `rbenv` and `ruby-build`
-        1. `rbenv install 2.2.3`
-        1. `rbenv global 2.2.3`
-        1. `rbenv rehash`
-    - C. `rvm`. A few folks have had more luck with rvm vs rbenv on linux.
-        1. Install rvm from https://rvm.io/
-        1. `rvm install 2.2.3`
-1. Install Node.js 0.12.4 and npm 2.10.1
-  1. Option A - nodesource repository
-    1. `curl -sL https://deb.nodesource.com/setup_0.12 | sudo -E bash -`
-    1. `sudo apt-get install -y nodejs`
-  1. Option B - Manual install
-    1. [Nodejs.org](https://nodejs.org/download/)
-1. Check that you have the correct versions of everything:
-  1. open a new Terminal window  
-  1. `ruby --version  # --> ruby 2.2.3`
-  1. `node --version  # --> v0.12.4`
-  1. `npm --version   # --> 2.10.1`
-
-#### Windows note: use an Ubuntu VM
-
-Many Windows developers have found that setting up an Ubuntu virtual machine is less painful than getting Ruby and other prerequisites running on Windows.
-
-* Option A: Use [VMWare Player](https://my.vmware.com/web/vmware/free#desktop_end_user_computing/vmware_player/4_0) and an [Ubuntu 14.04 iso image](http://releases.ubuntu.com/14.04.2/ubuntu-14.04.2-desktop-amd64.iso)
-* Option B: Use vagrant ([install](https://docs.vagrantup.com/v2/installation/)):
-  1. First clone the code.org git repo to get the provided Vagrantfile (you will be able to skip step 1 of the common setup instructions): `git clone https://github.com/code-dot-org/code-dot-org.git`
-  1. `cd code-dot-org`
-  1. `vagrant up`
-  1. `vagrant ssh`
-  1. Goto step 2 of the common setup instructions
-* Option C: Use AWS EC2: [launch Ubuntu 14.04 AMI](https://console.aws.amazon.com/ec2/home?region=ap-northeast-1#launchAmi=ami-d9fdddd8)
-
-## Common setup
-
-1. `git clone https://github.com/code-dot-org/code-dot-org.git`
-1. `sudo gem install bundler -v 1.10.6`
-1. `rbenv rehash` (if using rbenv)
-1. `cd code-dot-org`
-1. `bundle install`
-1. `rake install`
-1. `rake build`
-1. `sudo chown -R $(whoami) $HOME/.npm` (do we still need this step when using nvm?)
+This [Setup](https://github.com/code-dot-org/code-dot-org/blob/staging/SETUP.md) document describes how to set up your workstation to develop for Code.org.
```

Comment:
Use a relative link so that it's valid no matter which branch you're viewing.
```
This [Setup](./SETUP.md) document describes how to set up your workstation to develop for Code.org.
```